bindings/xmtp_rust_swift/src/subscriptions.rs

```diff
@@ -0,0 +1,30 @@
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use xmtp_networking::grpc_api_helper::Subscription;
+use xmtp_proto::xmtp::message_api::v1::Envelope;
+
+// Define the global HashMap for storing Subscriptions
+lazy_static! {
+    static ref SUBSCRIPTIONS: Mutex<HashMap<String, Arc<Subscription>>> =
```

Comment:
There must be an easier way to get access to mutable state, but the ChatGPT endorsed solution seems to work.